pom.xml

```diff
@@ -474,6 +475,37 @@
                         <scalaVersion>${scala.version}</scalaVersion>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <!-- 3.0.0-M4 was released 2019-11-13 -->
+                    <version>3.0.0-M4</version>
+                    <dependencies>
+                        <dependency>
+                            <groupId>org.apache.maven.surefire</groupId>
+                            <artifactId>surefire-junit4</artifactId>
+                            <version>3.0.0-M4</version>
+                        </dependency>
+                    </dependencies>
+                    <configuration>
+                        <includes>
+                            <include>**/*Test.java</include>
+                        </includes>
+                        <argLine>${maven.test.jvmargs}</argLine>
+                        <systemPropertyVariables>
+                            <java.net.preferIPv4Stack>true</java.net.preferIPv4Stack>
+                            <java.awt.headless>true</java.awt.headless>
+                            <user.timezone>UTC</user.timezone>
+                            <user.language>en</user.language>
+                            <user.region>US</user.region>
+                            <geomesa.scan.ranges.target>500</geomesa.scan.ranges.target>
+                            <geomesa.accumulo.test.tablet.servers>1</geomesa.accumulo.test.tablet.servers>
+                            <geomesa.hbase.test.region.servers>1</geomesa.hbase.test.region.servers>
+                        </systemPropertyVariables>
+                        <!-- these can interfere with the mini clusters we spin up for unit tests -->
+                        <excludedEnvironmentVariables>HADOOP_HOME,ZOOKEEPER_HOME,ACCUMULO_HOME,HBASE_HOME</excludedEnvironmentVariables>
```

Comment:
Not something for now, but it would be somewhat amusing if we found a good way to provide configuration to point a unit test suite at running cluster.  *shrugs*

Reply:
that wouldn't be very complicated tbh. i've done it before by just hand-jamming in values for MiniCluster.zookeepers, etc, but we could make it configurable easily enough.

Reply:
Well, if you need an idea for what to get me for Christimas in July...